src/offline/wallet.ts

```diff
@@ -0,0 +1,88 @@
+import {fromSeed} from 'bip32'
```

Comment:
We're getting rid of the `offline` folder, so we actually want this to be under `src` instead of in `offline`.

Reply:
Sure, I'll move it to be under `src`.